bootstrap.sh

```
@@ -1 +1,21 @@
 #!/bin/bash
+
+kind create cluster --config cluster.yml
```

Add robust failure handling and prerequisites checks at the top. Put `set -euo pipefail` after the shebang and check for required commands (kind, kubectl). This makes the script fail fast on errors and is important for reliable automation. Example additions: `set -euo pipefail` and checks like `command -v kind >/dev/null || { echo "kind not found"; exit 1; }`.